src/Microsoft.OData.Core/UriParser/SemanticAst/ExpandedNavigationSelectItem.cs

```diff
@@ -108,6 +108,7 @@ public ExpandedNavigationSelectItem(ODataExpandPath pathToNavigationProperty, IE
         /// <param name="searchOption">An levels clause for this expand (can be null)</param>
         /// <param name="levelsOption">An levels clause for this expand (can be null)</param>
         /// <param name="computeOption">A compute clause for this expand (can be null)</param>
+        /// <param name="applyOption">An apply clause for this expand (can be null)</param>

```

Comment:
>/// <param name="applyOption">An apply clause for this expand (can be null)</param> [](start = 8, length = 83)

Seems a lot of changes unrelated to the Large object streaming. Let me create a separate PR to fix them ahead.

Reply:
I merged the latest changes from master.

---
In reply to: [258748275](https://github.com/OData/odata.net/pull/1204#discussion_r258748275) [](ancestors = 258748275)